Source/WebCore/html/canvas/WebGLShader.h

```diff
@@ -52,7 +54,9 @@ class WebGLShader final : public WebGLSharedObject {
 
     GCGLenum m_type;
     String m_source;
+#if !USE(ANGLE)
     bool m_isValid;
```

Comment:
You might initialise this to false, so then you avoid one ifdef chunk in the constructor